pydrill/client/result.py

```diff
@@ -1,5 +1,26 @@
 # -*- coding: utf-8 -*-
 from pydrill.exceptions import ImproperlyConfigured
+import logging
+import re
+
+logger = logging.getLogger('pydrill')
+
+drill_pandas_type_map = {
+        'BIGINT': 'int64',
+        'BINARY': 'object',
+        'BIT':  'boolean',
+        'DATE': 'datetime64',
+        'FLOAT4': 'float32',
+        'FLOAT8': 'float64',
+        'INT': 'int32',
+        'INTERVALDAY': 'object',
+        'INTERVALYEAR': 'object',
+        'SMALLINT': 'int32',
+        'TIME': 'timedelta64',
+        'TIMESTAMP': 'datetime64',
+        'VARDECIMAL': 'object',
+        'VARCHAR' : 'string'
+        }
```

Comment:
How are multidimensional types such as `LIST` and `MAP` handled?

Reply:
Any data of a type that is not present in `drill_pandas_type_map` is left untouched by the new code, so their handling remains as it was.  If a better mapping for them is, or becomes, available then it can be added...

Separately I just tested with a pre-1.0.0 Pandas and I've had to add a fallback from the `string` dtype to `object` for Drill's `VARCHAR` to ensure compatibility.